tomls/omnibus-base-mainnet-andromeda/perps/arb-invokes.toml

```diff
@@ -0,0 +1,149 @@
+[setting.perpsArbMarketId]
```

Comment:
I believe we should have a "package" "standard-perps-market" and provision it with all settings in one simple step. As all of the standard markets are copypasted now